iron-fit-behavior.html

```diff
@@ -501,29 +491,35 @@
     },
 
 
-    __getPosition: function(hAlign, vAlign, size, positionRect, fitRect) {
+    __getPosition: function(hAlign, vAlign, rect, positionRect, fitRect) {
+      var margin = this._fitInfo.margin;
+      // Consider the margin as part of the size for position calculations.
+      var size = {
```

Comment:
Or I could keep the old signature, since the new one isn't any better compared to the previous one.